src/app/components/form-fields/dropdown/dropdown.component.html

```diff
@@ -1,10 +1,10 @@
-<label for="{{fieldId}}">{{label}}</label>
+<label>{{label}}</label>
```

Comment:
With the `aria-label` property defined on the next input ... do you still need a label ?